translations/monero-core_de.ts

```diff
@@ -2371,6 +2372,7 @@ Für den Fall mit Sendenachweis muss die Empfängeradresse nicht angegeben werde
     <message>
         <location filename="../wizard/WizardMemoTextInput.qml" line="65"/>
         <source>Seed copied to clipboard</source>
+        <translatorcomment>Here or before copying should be a warning. Coyping the seed to clipboard is a security risk.</translatorcomment>
```

Comment:
Don't add such things as translator comments. Create an issue or this will get lost.

Reply:
Yes, you are right.. I just added GH-1550